root/usr/lib64/ruby/vendor_ruby/discovery.rb

```diff
@@ -158,7 +158,7 @@ def write_tui result = 'success', code, body
 
 def upload(uri = discover_server, type = proxy_type, custom_facts = {})
   unless uri
-    log_err "Could not determine Foreman instance, add foreman.url or proxy.url kernel command parameter"
+    log_err "Could not determine instance type, add foreman.url or proxy.url kernel command parameter"
```

Comment:
As @ohadlevy  mentioned on irc we still have foreman.url and proxy.url here and we need them to determine the instance type.
Should we change it? just the `foreman.url`? keep it until the i18n change?